Mlem/App/Utility/Extensions/Content Models/Community1Providing+Extensions.swift

```diff
@@ -64,6 +64,24 @@ extension Community1Providing {
         }
     }
     
+    func toggleBlocked(feedback: Set<FeedbackType>) {
+        if feedback.contains(.toast) {
+            if !blocked {
```

Comment:
*nit* Can be more concise, also maybe like 1 cycle faster to check blocked first

```suggestion
        if !blocked, feedback.contains(.toast) {
```